File: CS/DXWebApplication26/App_Start/DashboardConfig.cs
using DevExpress.DashboardWeb;
using DevExpress.DashboardWeb.Mvc;
using System.Configuration;
using System.Web.Routing;

namespace DXWebApplication26
{
    public static class DashboardConfig {
        public static void RegisterService(RouteCollection routes) {
            routes.MapDashboardRoute("dashboardControl");

            DashboardFileStorage dashboardFileStorage = new DashboardFileStorage("~/App_Data/Dashboards");
            DashboardConfigurator.Default.SetDashboardStorage(dashboardFileStorage);

            DashboardConfigurator.Default.SetConnectionStringsProvider(
                new ConnectionStringsStorage(ConfigurationManager.ConnectionStrings["ConnectionsStorage"].ConnectionString));
        }
    }
}

File: VB/DXWebApplication26/App_Start/DashboardConfig.vb
Imports Microsoft.VisualBasic
Imports DevExpress.DashboardWeb
Imports DevExpress.DashboardWeb.Mvc
Imports System.Configuration
Imports System.Web.Routing

Namespace DXWebApplication26
	Public NotInheritable Class DashboardConfig
		Private Sub New()
		End Sub
		Public Shared Sub RegisterService(ByVal routes As RouteCollection)
			routes.MapDashboardRoute("dashboardControl", "DefaultDashboard")

			Dim dashboardFileStorage As New DashboardFileStorage("~/App_Data/Dashboards")
			DashboardConfigurator.Default.SetDashboardStorage(dashboardFileStorage)

			DashboardConfigurator.Default.SetConnectionStringsProvider(New ConnectionStringsStorage(ConfigurationManager.ConnectionStrings("ConnectionsStorage").ConnectionString))
		End Sub
	End Class
End Namespace